docs/deployment/heroku.md

```diff
@@ -0,0 +1,76 @@
+# Deploy to Heroku
+We designed Errbit to work well with Heroku. These instructions should result
+in a working deploy, but you should modify them to suit your needs:
+
+## Clone and prepare the source code repository
+```bash
+git clone git@github.com:errbit/errbit.git
+cd errbit
+```
+
+- Update `db/seeds.rb` with admin credentials for your initial login
+
+Commit the results:
+```bash
+git commit -m "Update db/seeds.rb with initial login"
+```
+
+## Create an app on Heroku and push the source code
+```bash
+gem install heroku
```

Comment:
Heroku doesn't use the gem version anymore. Installing the gem will result in a deprecation message: 
```
 !    The `heroku` gem has been deprecated and replaced with the Heroku Toolbelt.
 !    Download and install from: https://toolbelt.heroku.com
 !    For API access, see: https://github.com/heroku/heroku.rb
```

Instead Heroku uses a command-line interface called 'heroku toolbelt' as described here: https://devcenter.heroku.com/articles/heroku-command